### ISSUE TITLE
conf/scylla:yaml: `workdir` optoin doesn't work, `workdir,W` does

### ISSUE THREAD

**Issue by kbr-** [scylladb/scylladb#11503]
As in the title: you cannot specify the workdir using the `workdir` key in conf/scylla.yaml, you have to use `workdir,W` for some reason.

**Comment by nyh** [comment]
The weird thing is that the command line option "--workdir" does work - we use it in test/cql-pytest/run.py. 
So why doesn't it work in yaml?

The default config file `conf/scylla.yaml` has the commented-out lines
```
# Directory where Scylla should store all its files, which are commitlog,
# data, hints, view_hints and saved_caches subdirectories. All of these
# subs can be overriden by the respective options below.
# If unset, the value defaults to /var/lib/scylla
# workdir: /var/lib/scylla
```

Which suggests that "workdir" either worked in the past or at least should have worked.

**Comment by michoecho** [comment]
Duplicate of #9500, which is a duplicate of #7478, meaning that this occurs on average once every 2000 issues.

**Comment by michoecho** [comment]
Posted a fix to #11506.

**Comment by avikivity** [comment]
Developer-oriented bug, and these live in master, so not backporting.
